example/alertmanager-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-main
+data:
+  alertmanager.yaml: |-
+    global:
+      resolve_timeout: 5m
+    route:
+      group_by: ['alertname']
```

Comment:
Yes, which makes more sense in practice. If something is wrong with a certain service, chances are more than one of its alerts are firing, which you still want to see as one comprehensive notification.